### PR TITLE
[GH-695] Upgraded ex_rlp to 0.3.0

### LINKED ISSUE #695: Upgrade ex_rlp to 0.3.0

### PR DESCRIPTION
**Pull request by DanielaIvanova**
resolves #695

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -52,6 +52,7 @@ defmodule EpochElixir.Mixfile do
       {:credo, "~> 0.9.3", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 0.5", only: [:dev, :test], runtime: false},
       {:excoveralls, "~> 0.8.1", only: :test},
+      {:plug_cowboy, "~> 1.0"},
```
Comment (thepiwo) [inline mix.exs]: why is this added?

Comment (DanielaIvanova) [reply mix.exs]: I couldn't start the project without plug_cowboy.

Comment (thepiwo) [reply mix.exs]: Do you have an idea why this wasn't needed before?

Comment (DanielaIvanova) [reply mix.exs]: Maybe, there are some problems with Phoenix.
